apps/nextjs-website/src/app/[productSlug]/tutorials/[...productTutorialPage]/page.tsx

```diff
@@ -71,18 +75,74 @@ const Page = async ({ params }: { params: Params }) => {
     },
   };
 
+  const { relatedLinks } = await getOverview(productSlug);
+  const { overview } = translations;
+
   return (
     <ProductLayout
       product={props.product}
       path={props.path}
       bannerLinks={props.bannerLinks}
       showBreadcrumbs={false}
     >
-      <EContainer>
-        <Box sx={{ padding: '56px 0' }}>
-          <GitBookContent content={props.body} config={props.bodyConfig} />
+      <FragmentProvider>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', lg: 'row' },
+            maxWidth: 'calc(822px + 64px + 270px)',
+            margin: '0 auto',
+          }}
+        >
+          <Box
+            sx={{
+              // 78px is the height of the header, 80px is the height of the product header, 24 is the title padding
+              paddingTop: 'calc(78px + 80px - 24px)',
+              flexGrow: { lg: 1 },
+              maxWidth: {
+                xs: '100%',
+                lg: '822px',
+              },
+            }}
+          >
+            <GitBookContent content={props.body} config={props.bodyConfig} />
+          </Box>
+          <Box
+            sx={{
+              display: { xs: 'none', lg: 'initial' },
+              position: 'relative',
+              // 78px is the height of the header, 80px is the height of the product header
+              paddingTop: 'calc(78px + 80px)',
```

Comment:
suggestion: can you create constants to keep track of fixed height values?
I think on "apps/nextjs-website/src/components/atoms/GuideMenu/GuideMenu.tsx" you can find something similar.

The best solution would be to export all menu heights to another config file and use them in each position when you need them